src/components/FormField/__snapshots__/FormField.test.tsx.snap

```diff
@@ -19,7 +19,9 @@ exports[`FormField should match snapshot 1`] = `
       data-testid="name"
       id="name"
       label="Name"
+      name="name"
       type="string"
+      value=""
```

Comment:
I just noticed `type` there, this can lead to several problem in future. Might not be related to form itself but we need to be careful with our types coming from the `prepare` api.